OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -21,16 +21,21 @@ namespace OpenRA.Mods.Common.LoadScreens
 {
 	public class BlankLoadScreen : ILoadScreen
 	{
-		public LaunchArguments Launch;
 		protected ModData ModData { get; private set; }
 
 		bool initialized;
+		LaunchArguments launch;
 
 		public virtual void Init(ModData modData, Dictionary<string, string> info)
 		{
 			ModData = modData;
 		}
 
+		public LaunchArguments GetLaunchArguments()
```

Comment:
Why not just a `get; private set;` instance property (like `ModData`)?

Reply:
> Interfaces cannot contain instance fields csharp(CS0525)